src/main/java/org/mockito/internal/junit/DefaultStubbingLookupListener.java

```diff
@@ -68,17 +69,15 @@ private static List<Invocation> potentialArgMismatches(
         List<Invocation> matchingStubbings = new LinkedList<>();
         for (Stubbing s : stubbings) {
             if (UnusedStubbingReporting.shouldBeReported(s)
-                    && s.getInvocation()
-                            .getMethod()
-                            .getName()
-                            .equals(invocation.getMethod().getName())
+                    && Objects.equals(
+                            s.getInvocation().getMethod().getName(),
+                            invocation.getMethod().getName())
                     // If stubbing and invocation are in the same source file we assume they are in
                     // the test code,
                     // and we don't flag it as mismatch:
-                    && !s.getInvocation()
-                            .getLocation()
-                            .getSourceFile()
-                            .equals(invocation.getLocation().getSourceFile())) {
+                    && !Objects.equals(
+                            s.getInvocation().getLocation().getSourceFile(),
```

Comment:
Was causing NPE here in case of `ProxyMockMaker` because `s.getInvocation().getLocation().getSourceFile()` is `null`